doc/devguide/extending/app-layer/transactions.rst

```diff
@@ -0,0 +1,112 @@
+************
+Transactions
+************
+
+Transactions are what guides when detecting and logging happen in Suricata. They’ll also help when dealing with protocols that can have large PDUs, like TCP, in controlling state for partial rule matching, for rules that mention more than one field.
```

Comment:
"Transactions are abstractions that help guide detection and loggin in Suricata"